### ISSUE TITLE
Rollover does not happen

### ISSUE THREAD

**Issue by muslimfaisal** [OS4ED/openSIS-Responsive-Design#138]
Hello, we are using v7.4 responsive. When we try and rollover, it says that it is saving the database, then goes to say school year rolling over, please wait...  But this goes on for hours with no effect whatsoever on the database. The new academic year is not created. 

In the release notes of v7.5 it does not say that this issue is fixed. Is anyone else facing this issue? 

Can we downgrade to v6.5 so that we can rollover? Is there another solution?

**Comment by openSISAdmin** [comment]
This is an isolated issue dependent on data, OS and server configuration. Please purchase support contract to get help.

Thanks

**Comment by EvanofMT** [comment]
I had a similar issue with v 7.6.  The rollover attempted to my a backup copy of the database and then stopped after rolling over the Users (ie. staff) table.

After version 8 was released, I copied the following files from version 8 and added them to my version 7.6.  Just tested and the rollover completed successfully.

* /functions/CustomFunctionsFnc.php
* /js/Ajaxload.js
* /modules/tools/Rollover.php
* /BackupForRollover.php
* /RolloverShadow.php

It seems like "RolloverShadow.php" is the problem.  I haven't compared the differences in between v8 and v7.6, but tested with the first four files in the list with no success.  After I copied over "RolloverShadow.php" the rollover completed successfully.
